src/components/MoneyTransferModal.vue

```diff
@@ -4,7 +4,7 @@
     <template v-if="step === 1">
       <div class="modal-card">
         <header class="modal-card-head">
-          <p class="modal-card-title is-title-shrink">Envoyer de l'argent</p>
+          <p class="modal-card-title is-title-shrink">Envoyer de l'argent - 1/2</p>
```

Comment:
I wonder if it is necessary to display this information here, because it would be logical for the user to have another modal to enter the amount to transfer

Reply:
Thanks for your input, and I partially agree with you, bu I'm really wary of what seems logical and what seems not when it comes to end-user understanding. I think we never make things explicit enough.

In the end I think we should make the PO (@MatSarr) decide what should be done here.

Reply:
i agree

Reply:
@MatSarr can you decide what should be done here ?
`Envoyer de l'argent - 1/2` then `Envoyer de l'argent - 2/2`
Or we keep `Envoyer de l'argent` for both steps ?

I'm really undecided here. Telling the step is user-friendly, not telling looks better ...

Reply:
Undecided too. Following sprint meeting, Stéphan said he prefer "Envoyer de l'argent - 1/2 then Envoyer de l'argent - 2/2" so let's do that